website/versioned_docs/version-2.6/index.md

```diff
@@ -1,25 +1,279 @@
 ---
-sidebar_label: 'Quick start'
-title: 'Quick start'
+sidebar_label: 'Getting started'
+title: 'Getting started'
 ---
 
 # Apify API client for JavaScript
 
-`apify-client` is the official library to access [Apify API](https://docs.apify.com/api/v2) from your
-JavaScript applications. It runs both in Node.js and browser and provides useful features like
-automatic retries and convenience functions that improve the experience of using the Apify API.
+`apify-client` is the official library to access the [Apify REST API](https://docs.apify.com/api/v2) from your
+JavaScript/TypeScript applications. It runs both in Node.js and browser and provides useful features like
+automatic retries and convenience functions that improve the experience of using the Apify API. All requests and responses (including errors) are encoded in JSON format with UTF-8 encoding.
 
-## Quick Start
+## Pre-requisites
+
+`apify-client` requires Node.js version 16 or higher. Node.js is available for download on the [official website](https://nodejs.org/). Check for your current node version by running:
+
+```bash
+node -v
+```
+
+## Installation
+
+You can install the client via [NPM](https://www.npmjs.com/) or use any other package manager of your choice.
+
+```bash
+npm i apify-client
+# or
+pnpm add apify-client
+# or
+yarn add apify-client
+# or
+bun add apify-client
+```
```

Comment:
nit: we can use a docusaurus plugin for this, which will turn it into tabs automatically (and we should be consistent, other places only show `npm i` i think)

Reply:
I used the tabs, and it looks much better, thanks for the tip! I wanted to keep all the examples - it's getting started (there should be as much options and explanations as possible)